EIPS/eip-3473.md

```diff
@@ -0,0 +1,463 @@
+---
+eip: 3473
+title: ERC-659 Multiple Callable Bonds Standard
+author: Yohji Sakamoto <info@sgm.finance>

```

Comment:
```suggestion
author: Yohji Sakamoto (@sgmfinance)
```
We require authors to include at least one github username in the authors field because it allows that author interact with the automerge bot for update to the EIP.